ONSdigital/eq-author-app
# Ear-1976-pipe-answers-into-answer-label

### What is the context of this PR?

This PR is linked to this [Jira ticket](https://jira.ons.gov.uk/browse/EAR-1976)

### How to review 



- Checkout `EAR-1976-pipe-answers-into-answer-label`
  - create a questionnaire with two question pages 
  - use basic answer types for each answer on both question pages 
  - check that the input field for the answer label is a rich text editor
  - check that there is validation for an empty text field
  - ensure that piping is disabled for the first answer label
- Check that piping is enabled for the second answer label and pipe in the answer from the above question
  - ensure that the preview page displays the correct data
  - navigate back to the design page
  - check that moving the piped answer to a lower level throws a validation error
  - check that deleting the piped answer throws a validation error
  - check for all basic answer types
- Ensure that the runner schema is correct when there is a piped answer
